src/v2/Apps/Fair/Routes/__tests__/FairArticles.jest.tsx

```diff
@@ -1,32 +1,51 @@
-import { graphql } from "react-relay"
-import { FairArticlesFragmentContainer } from "../FairArticles"
+import {
+  FairArticlesPaginationContainer,
+  FAIR_ARTICLES_QUERY,
+} from "../FairArticles"
 import { setupTestWrapper } from "v2/DevTools/setupTestWrapper"
 
 jest.unmock("react-relay")
 
 const { getWrapper } = setupTestWrapper({
-  Component: FairArticlesFragmentContainer,
-  query: graphql`
-    query FairArticles_Test_Query {
-      fair(id: "example") {
-        ...FairArticles_fair
-      }
-    }
-  `,
+  Component: FairArticlesPaginationContainer,
+  query: FAIR_ARTICLES_QUERY,
+  variables: {
+    id: "example",
+    first: 10,
+  },
 })
 
 describe("FairArticles", () => {
-  it("renders correctly", () => {
+  it("renders the articles", () => {
     const wrapper = getWrapper({
+      ArticleConnection: () => ({ totalCount: 1 }),
       Article: () => ({ title: "Example Article" }),
       Author: () => ({ name: "Example Author" }),
     })
 
     expect(wrapper.find("h1")).toHaveLength(1)
 
+    // FIXME: Unable to mock the contents of the connection
+    // when using a pagination container
+    // const html = wrapper.html()
+    // expect(html).toContain("Example Article")
+    // expect(html).toContain("Example Author")
```

Comment:
This is weird. I'm leaning towards this being a bug in possibly one of our dependencies because of how the fields are `undefined` by default instead of the placeholder values that the mock generator usually returns. But I might be missing some mysterious configuration detail somewhere.

I'm able to mock out the connection `totalCount`, but not the articles field directly, nor does mocking the type itself work. Switching the component to a fragment container allows the mocks to work fine. Will make a note to try build a simple replication.

Reply:
(This test is pretty low-value anyway so I'm fine with this TODO.)